tools/llm_bench/task/audio_generation.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023-2024 Intel Corporation
```

Comment:
not sure that file name is correct. Audio generationt means that pipeline generates audio as output result, but you have opposite - uses audio as input and produces text. correct task name is spechh_to_text or automatic speech recogntion